### PR TITLE
fixed mispelling of funereal to funeral

### LINKED ISSUE #2193: Funereal/Funeral Melody Mismatch
**Describe the bug**
The melody is called `Funereal` in blocks, but is called `Melodies.Funeral` in JavaScript. 

**Expected behavior**
The two should match

image

image

### PR DESCRIPTION
**Pull request by hristoramirez**
fixes #2193

### CODE REVIEW

File: libs/core/_locales/core-strings.json [hunk 1]
```diff
@@ -155,7 +155,7 @@
   "Melodies.Chase|block": "chase",
   "Melodies.Dadadadum|block": "dadadum",
   "Melodies.Entertainer|block": "entertainer",
-  "Melodies.Funeral|block": "funereal",
+  "Melodies.Funeral|block": "funeral",
```
Comment (darzu) [inline libs/core/_locales/core-strings.json:158]: Nice job!

**Comment by abchatra** [comment]
Is there a backward compat issue here? @riknoll ?

**Comment by riknoll** [comment]
It will break translations, that's all

**Comment by riknoll** [comment]
But also, isn't funereal technically correct? Maybe worth fixing just because it *looks* wrong even though it is a word

**Comment by darzu** [comment]
Oh wow, I had no idea funereal was a real word. It was "funeral" in JS, so maybe one was a typo?

**Pull request merged**